src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -5,6 +5,8 @@
 #include "orbitmainwindow.h"
 
 #include <absl/flags/declare.h>
+#include <qmenubar.h>
+#include <qobject.h>
```

Comment:
Nit

Reply:
One day I will find time to setup auto fixing iwyu on every save. This gets really annoying. Thank you for pointing it out. done (and rebased)